Formula/abyss.rb

```diff
@@ -0,0 +1,30 @@
+class Abyss < Formula
+  desc "Genome sequence assembler for short reads"
+  homepage "http://www.bcgsc.ca/platform/bioinfo/software/abyss"
+  url "https://github.com/bcgsc/abyss/releases/download/2.0.2/abyss-2.0.2.tar.gz"
+  sha256 "d87b76edeac3a6fb48f24a1d63f243d8278a324c9a5eb29027b640f7089422df"
```

Comment:
Are `head` being intentionally removed?

Reply:
Yeah, 0 uses of it in the last 90 days.

Reply:
I use `--HEAD` as a mode for a user to test a fix of a bug that's affecting them before tagging a release. No bugs in the last ninety days would mean no installs of `--HEAD` in the last ninety days.

Reply:
It's also 0 at 30, 60, 90, 120, 150, 180, 210, 240, ...

Reply:
My preference is to include `head`. Final call is yours of course.

Reply:
Sure let's include it. But just this once since you're upstream :p